resources/daemonset.go

```diff
@@ -0,0 +1,104 @@
+package resources
+
+import (
+	"errors"
+	"log"
+
+	"k8s.io/kubernetes/pkg/apis/extensions"
+	"k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+//DaemonSet is wrapper for K8s DaemonSet object
+type DaemonSet struct {
+	DaemonSet *extensions.DaemonSet
+	Client    unversioned.DaemonSetInterface
+}
+
+func daemonSetKey(name string) string {
+	return "daemonset/" + name
+}
+
+func daemonSetStatus(d unversioned.DaemonSetInterface, name string) (string, error) {
+	daemonSet, err := d.Get(name)
+	if err != nil {
+		return "error", err
+	}
+	if daemonSet.Status.CurrentNumberScheduled == daemonSet.Status.DesiredNumberScheduled {
```

Comment:
Current number scheduled returns:
```
// CurrentNumberScheduled is the number of nodes that are running at least 1
// daemon pod and are supposed to run the daemon pod.
```
but if you prefer i can check each pod separately.